vendor/cmdliner/LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011 Daniel C. Bünzli
+Copyright (c) 2011 The cmdliner programmers
```

Comment:
```suggestion
Copyright (c) 2011 The `Cmdliner` Programmers
```

Since `Cmdliner` is capitalised in the code, perhaps it would be good to be capitalized and in monospace here. If you do, perhaps capitalise Programmers too.

If you choose to do this, I will go through and suggest on all of them, so you can just accept suggestion. Let me know! :)

Reply:
Everything under `vendor/` is pulled from third party code so we can't change it.